app/bundles/MarketplaceBundle/Views/Package/list.html.twig

```diff
@@ -0,0 +1,148 @@
+{% set isIndex = tmpl == 'index' ? true : false %}
+{% set tmpl = 'list' %}
+
+{% extends isIndex ? 'MauticCoreBundle:Default:content.html.twig' : 'MauticCoreBundle:Default:raw_output.html.twig' %}
+{% block mauticContent %}Package{% endblock %}
+{% block headerTitle %}{{ 'marketplace.title'|trans|raw }}{% endblock %}
```

Comment:
I'd use htmlpurifier here too. It's unlikely, but translations are community-powered and an attacker can place some JS into a translation of a specific language translation.